/mouse_select_point:
 ros__parameters:
  # pcd_folder: "/home/ww/datasets/PanZhiHua/9.bag/alldata/"
  pcd_folder: "/home/ww/datasets/LvShun_ros2/perception/1011obs-4_2/all/"
  save_annotation_radar_flag: true
  # file_name_annotationed: "/home/ww/datasets/PanZhiHua/9.bag/ann/"
  file_name_annotationed: "/home/ww/datasets/LvShun_ros2/perception/1011obs-4_2/ann/"
  file_index: 10
  lidar_cluster_flag: false
  radar_track_flag: true
  


/radar_auto_annotation:
  ros__parameters:
    radar2camera_rot: [ 1.0,  0.0,   0.0,
                        0.0,  0.0,  1.0,
                        0.0,  1.0,   0.0]
    radar2camera_trans: [ -0.19, 0.09, 0.00]

    camera_intrinsic: [ 695.525, 0.00449423, 326.046,
                   0.0,     696.099,    235.763,
                   0.0,     0.0,        1.0 ]
                   

    # PanZhiHua
    # radar2lidar_rot: [  0.999954, -0.0070342, -0.00481069,
    #                     -0.00516621, 0.995131, -0.00063,
    #                     0.00677553, 0.0000851, 0.99514776 ]

    # radar2lidar_trans: [0.0, 0.0, -0.04]


    radar2lidar_rot: [ 0.0, 1.0, 0.0,
                        1.0, 0.0, 0.0,
                        0.0, 0.0, 1.0 ]
    radar2lidar_trans: [0.0, 0.0, -0.06]